Articles/Setup/Install/install-umbraco-with-templates.md

```diff
@@ -0,0 +1,97 @@
+---
+v8-equivalent: none
+versionFrom: 9.0.0
+verified-against: alpha-4
+state: partial
+updated-links: false
+---
+
+# Install Umbraco with .NET CLI
+We have made custom Umbraco templates that are available for use with `dotnet new`. The steps below will demonstrate the minimum amount of actions required to get you going and set up an Umbraco project from the command line using .NET Core templates.
+
+## Install the template
+1. Install the latest [.NET SDK](https://dotnet.microsoft.com/download).
+
+1. Run `dotnet new -i Umbraco.Templates::*` to install the project templates.  
+*The solution is packaged up into the NuGet package [Umbraco.Templates](https://www.nuget.org/packages/Umbraco.Templates) and can be installed into the dotnet CLI*.
```

Comment:
The nuget link is dead, but I assume this will be up and running once v9 is released

Reply:
Once again, I have mentioned this already :) 
> Update the Umbraco.Templates nuget package link once we publish it in install-umbraco-with-templates.md;
It is currently referenced as https://www.nuget.org/packages/Umbraco.Templates;